shared/actions/config/index.tsx

```diff
@@ -420,12 +420,22 @@ const routeToInitialScreen = (state: Container.TypedState) => {
   }
 }
 
+let maybeLoadAppLinkOnce = true
 const maybeLoadAppLink = (state: Container.TypedState) => {
   const phones = state.settings.phoneNumbers.phones
   if (!phones || phones.size > 0) {
     return
   }
 
+  if (
+    !maybeLoadAppLinkOnce ||
```

Comment:
and this should be true, just for consistency with the initial routing. my snippet was wrong